internal/pkg/skuba/replica/testdata/fake.go

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
I might place the mocks, helpers into `internal/pkg/skuba/replica/testing.go` file.
Reference: https://medium.com/@povilasve/go-advanced-tips-tricks-a872503ac859

Reply:
A bit confused, the link doesn't talk about testing.go. 
I've placed the fake file in nest directory so it's on a package `testdata` instead of using package `replica`. This is to separate the vars used for testings.

Reply:
Never mind me, just saw the part.

Reply:
Seems its suggesting to place the vars to testing.go under different package not the same package to `replica`. This is the same approach just using `fake.go` instead of `testing.go`.
Am I missing anything?

Reply:
Make sense after discussion.

Reply:
Thanks